xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamBluray.cpp

```diff
@@ -332,7 +332,7 @@ bool CDVDInputStreamBluray::Open()
     m_navmode = false;
     m_titleInfo = GetTitleLongest();
   }
-  else if (resumable && m_item.GetStartOffset() == STARTOFFSET_RESUME)
+  else if (resumable && m_item.GetStartOffset() == STARTOFFSET_RESUME && m_item.IsResumable())
```

Comment:
Just curious why an item can have a startoffset resume and not be resumable?

Reply:
Look what I wrote in the PR description. This special value indicates that we want to resume only, not that there is actually a resume point set.

Reply:
Just a curious question, not judging...